src/Mapping/PropertyMappingResolver.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vich\UploaderBundle\Mapping;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Vich\UploaderBundle\Exception\MappingNotFoundException;
+use Vich\UploaderBundle\Naming\ConfigurableInterface;
+use Vich\UploaderBundle\Util\ClassUtils;
+
+/**
+ * PropertyMappingResolver.
+ *
+ * @author Dustin Dobervich <ddobervich@gmail.com>
```

Comment:
Please add your references here. If you prefer not be mentioned, remove this line

Reply:
The changes are minimal, so I felt ashamed adding my name up there, but if you insist ;).

Reply:
Reverted back to no reference.